### PR TITLE
Webui Commands

### PR DESCRIPTION
**Pull request by mappum**
This PR adds a couple commands that are used by the webui. `ipfs log read` streams the output of the event log, and `ipfs config replace <path>` replaces the config file with a different file.

### CODE REVIEW

File: core/commands/config.go [hunk 1]
```diff
@@ -183,3 +215,12 @@ func editConfig(filename string) error {
 	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
 	return cmd.Run()
 }
+
+func replaceConfig(r repo.Repo, file io.Reader) error {
+	var cfg config.Config
+	if err := json.NewDecoder(file).Decode(&cfg); err != nil {
+		return errors.New("Failed to decode file as config")
```
Comment (jbenet) [inline core/commands/config.go]: :+1: glad it's validated some

File: core/commands/log.go [hunk 2]
```diff
@@ -50,3 +68,64 @@ output of a running daemon.
...
 }
+
+var logReadCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Read the logs",
+		ShortDescription: `
+'ipfs log read' is a utility command used to read the log output.
+
+By default, the last 50 lines are returned. For more, use '-n=<number of lines>'.
+Use '--stream' (or '-s') to stream the current output (in addition to the '-n' lines).
```
Comment (jbenet) [inline core/commands/log.go]: Let's rename this to `tail` and `tail -f` (to follow) as users are used to that.

```
ipfs log tail
ipfs log tail -f
```

**Comment by jbenet** [comment]
LGTM! :+1: 

(would be nice if we can vendor the files as a separate commit beforehand, just to make it easier to read what changed in our code)

**Comment by mappum** [comment]
>(would be nice if we can vendor the files as a separate commit beforehand, just to make it easier to read what changed in our code)

ACK, will do in the future.

**Comment by mappum** [comment]
>Let's rename this to tail and tail -f (to follow) as users are used to that.

I had actually forgotten to edit the helptext, but the command only follows now (`-f`), it doesn't show previous lines. That's because something janky was going on and most of the time we couldn't get the previously saved output (might just be this tail library). We can come back to that later if it's necessary.

**Comment by jbenet** [comment]
:+1:

**Pull request merged**